index.html

```diff
@@ -1,13 +1,250 @@
+                <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                  <span class="info-text">Back End Dev</span></li>
+                <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                  <span class="info-text">2015</span></li>
+              </ul>
+        
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+          
+          <ul class="tech-list">
+            <li class="project-tech">html</li>
+            <li class="project-tech">css</li>
+            <li class="project-tech">javascript</li>
+          </ul>
+            <button class="outlined-button">See Project</button>
+        
+        </div>
+    </div>
+</section>
+<section class="portfolio">
+  <div class="project">
+      <img src="./assets/tonic-girl.png" alt="" class="project-preview">
+
+      <div class="project-info">
+          <h2 class="h-text project-name">Tonic</h2>
+            <ul class="skills-list">
+              <li class="checkbox"><span class="info-text">CANOPY</span></li>
+              <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">Back End Dev</span></li>
+              <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">2015</span></li>
+            </ul>
+        
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
+      </div>
+  </div>
+</section>
+
+<section class="portfolio test2">
+  <div class="project">
+      <img src="./assets/multips.png" alt="" class="project-preview">
+
+      <div class="project-info">
+          <h2 class="h-text project-name">Tonic</h2>
+            <ul class="skills-list">
+              <li class="checkbox"><span class="info-text">CANOPY</span></li>
+              <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">Back End Dev</span></li>
+              <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">2015</span></li>
+            </ul>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
```

Comment:
- Like the previous reviewer suggested, please implement the hover styles for all your buttons. Kindly add them as specified by the Figma design. Here is a copy of the design palette 👇
Design template:
<img width="326" alt="interactions" src="https://user-images.githubusercontent.com/73607512/219898738-066bbed2-528b-480a-85f7-6ba8b9004f20.PNG">